main.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
+using std::ofstream;
+using std::ifstream;
+
+int main(int argc, char *argv[]) {
+
+
+    string name = "";
+    string email = "";
+    string cypher = "";
+    string timezone_offset = "";
+    string file_path = "";
+    string init_edit = "";
+
+    cout << "Enter 'init' or 'edit': ";
+    cin >> init_edit;
+
+    if(static_cast<string>(argv[1]) == "init")
+    {
+        while (name == "")
+        {
+            cout << "Enter first and last name: ";
+            getline(cin, name);
+        }
+        while (email == "" )
+        {
+            cout << "Enter e-mail address: ";
+            getline(cin, email);
+        }
+
+        while (cypher == "")
+        {
+            cout << "Enter cypher: ";
+            getline(cin, cypher);
+        }
+
+        while (timezone_offset == "")
+        {
+            cout << "Enter timezone offset: ";
+            getline(cin, timezone_offset);
+        }
+
+        cout << "Enter path to known recipients file: ";
+        getline(cin, file_path);
+
+        if (file_path == "")
+        {
+            file_path = "knownrecipients.txt";
+        }
+
+        cout << file_path << endl;
+
+        ofstream init_output;
+        init_output.open (file_path);
+
+        init_output << name << endl;
+        init_output << email << endl;
+        init_output << cypher << endl;
+        init_output << timezone_offset << endl;
+        init_output << file_path << endl;
+
+        init_output.close();
+    }
+    else if (static_cast<string>(argv[2]) == "edit")
```

Comment:
this is still `argv[1]` here
`argv[2]` would identify the field in the file that you are editing in this mode